core/src/main/java/discord4j/core/spec/EmbedCreateSpec.java

```diff
@@ -86,7 +86,17 @@ public EmbedCreateSpec setTimestamp(Instant timestamp) {
      * @return This spec.
      */
     public EmbedCreateSpec setColor(final Color color) {
-        requestBuilder.color(color.getRGB() & 0xFFFFFF);
+        setColor(color.getRGB());
```

Comment:
@darichey found that this won't compile. Also this PR is not going through circle..

Reply:
The line was just missing a `return`. cdb7109 should compile.